### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by JeroenSfdc**
IMHO `private static String getNameUniqueField` can be simplified.

### CODE REVIEW

**Comment by ericrsmith35** [comment]
Thank you.  I will review.

**Comment by alexed1** [comment]
Thanks for the contribution, Jeroen. Can you summarize your changes?

Also fyi i just invited you to the Flow Projects slack workspace for flow developers

**Comment by JeroenSfdc** [comment]
> Thanks for the contribution, Jeroen. Can you summarize your changes?
> 
> Also fyi i just invited you to the Flow Projects slack workspace for flow developers

- Some slight stylish changes (see diffs)
- While adding test coverage for the `getUniqueNameField`, while reviewing the code I came to conclusion that the method has unreachable code. Every object AFAIK today has a field with the `isNameField` property (whether that be `Name` or `CaseNumber` or anything else). But I might be wrong. 
(I have separate PR for added test coverage for `ers_DataTableController` (apols for not bundling in single PR...)) 
https://github.com/alexed1/LightningFlowComponents/pull/1008

File: flow_screen_components/datatable/force-app/main/default/classes/ers_DatatableController.cls [hunk 1]
```diff
@@ -437,54 +441,62 @@ public with sharing class ers_DatatableController {
...
         DescribeSObjectResult objDescribe = sobjType.getDescribe();
         Map<String, Schema.SObjectField> fieldMap = objDescribe.fields.getMap();
+
         for(String fieldName : fieldMap.keySet()) {
             SObjectField objField = fieldMap.get(fieldName);
             Schema.DescribeFieldResult dfr = objField.getDescribe();
             if(dfr.isNameField()) {
                 strResult = dfr.getName();
                 break;
             }
```
[the remark this answers is not in the document]
Comment (ericrsmith35) [reply flow_screen_components/datatable/force-app/main/default/classes/ers_DatatableController.cls:457]: I am a bit nervous about removing code when something is currently working.  Can we be certain that Salesforce doesn't have some strange standard object out there where there isn't a valid value for isNameField?

**Comment by ericrsmith35** [comment]
Incorporated in v3.4.5
